docs/en/ingest-management/elastic-agent/configuration/outputs/output-elasticsearch.asciidoc

```diff
@@ -0,0 +1,586 @@
+:output-type: elasticsearch
+
+[[elasticsearch-output]]
+== {es} output
+
+The `elasticsearch` output sends events directly to {es} by using the
+{es} HTTP API.
+
+*Compatibility:* This output works with all compatible versions of {es}. See the
+https://www.elastic.co/support/matrix#matrix_compatibility[Elastic Support
+Matrix].
+
+Example configuration:
+
+//TODO: Provide a example that shows more of the settings users are likely to
+//change.
+
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["https://myEShost:9200"] <1>
+----
+<1> To enable SSL, add `https` to all URLs defined under __hosts__.
+
+The `elasticsearch` output supports the following settings, grouped by category.
+Many of these settings have sensible defaults that allow you to run {agent} with
+minimal configuration.
+
+<<output-elasticsearch-commonly-used-settings>>::
+Settings frequently changed for this input type.
+
+<<output-elasticsearch-authentication-settings>>::
+Settings for authenticating with {es}.
+
+<<output-elasticsearch-data-parsing-settings>>::
+Settings used to parse, filter, and transform data.
+
+<<output-elasticsearch-http-settings>>::
+Settings that modify the HTTP request sent to {es}.
+
+<<output-elasticsearch-performance-tuning-settings>>::
+Settings that control timeouts and other behavior that affects performance.
+
+
+//QUESTION: Do we still support all the ways to authenticate against ES, or
+//is token-based authentication the only way?
+
+[[output-elasticsearch-commonly-used-settings]]
+=== Commonly used settings
+
+[cols="2*<a"]
+|===
+| Settings | Description
+
+//QUESTION: I've removed index and indices because I don't think it makes sense
+//to document these settings for agent. Let me know if that's not correct.
+
+// =============================================================================
+
+// tag::enabled-setting[]
+|
+[id="output-{output-type}-enabled-setting"]
+`enabled`
+
+| (boolean) Enables or disables the output. If set to `false`, the output is
+disabled.
+
+*Default:* `enabled`
+
+// end::enabled-setting[]
+
+// =============================================================================
+
+// tag::escape_html-setting[]
+
+//QUESTION: What category would you put this in? Didn't fit in any other, so I
+//put it here.
```

Comment:
I think 'Data parsing, filtering, and manipulation settings' is perfect for this.  I'll make suggestions to add / remove ti.